src/skills_traits.hpp

```diff
@@ -9,6 +9,8 @@
 #ifndef BoE_SKILLS_TRAITS_HPP
 #define BoE_SKILLS_TRAITS_HPP
 
+#include <iostream>
```

Comment:
We don't want to include `<iostream>` in a header. Try `<iosfwd>` instead. The associated source file may need to add `<ostream>` and/or `<istream>`.

Reply:
good idea